package.json

```diff
@@ -6,10 +6,18 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@graphql-codegen/cli": "^3.3.1",
+    "@graphql-codegen/client-preset": "^3.0.1",
+    "@graphql-codegen/typescript": "^3.0.4",
+    "@graphql-codegen/typescript-document-nodes": "^3.0.4",
+    "@graphql-codegen/typescript-operations": "^3.0.4",
+    "@graphql-codegen/typescript-urql": "^3.7.3",
+    "dotenv": "^16.0.3",
     "husky": "^8.0.3"
   },
   "scripts": {
     "build": "yarn workspace @artblocks/contracts run build && yarn workspace @artblocks/sdk run build",
+    "codegen:project": "graphql-codegen dotenv_config_path=./.env --require dotenv/config --config graphql.config.ts --project",
```

Comment:
can you add the new env variables to a `.env.example` file?

Reply:
Done!